src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineLinuxLauncher.java

```diff
@@ -130,7 +130,7 @@ protected void launch(ComputeEngineComputer computer, TaskListener listener, Ins
 
 
             //TODO: allow jvmopt configuration
-            String launchString = "java -jar " + tmpDir + "/agent.jar";
+            String launchString = "java -jar " + jenkinsDir + "/agent.jar";
```

Comment:
Maybe use `File.separator`?

Reply:
Does that work for both Operating systems?

Reply:
I think the behavior is based on whatever OS Java is currently running on

Reply:
Thats the whole idea :)

Reply:
I see, so are we assuming the master (running the JVM) will be same OS as the agents? Either way, are we suggesting we go with File.separator instead?

Reply:
Ok.. no.. :)
Its for sure not true..
So if we dont have `remote separator` we should leave it as is..